core/storage/classifier/gae_models.py

```diff
@@ -63,9 +63,6 @@ class ClassifierTrainingJobModel(base_models.BaseModel):
     # It is incremented by TTL when a job with status NEW is picked up by VM.
     next_scheduled_check_time = ndb.DateTimeProperty(required=True,
                                                      indexed=True)
-    # The classifier data which will be populated when storing the results of
-    # the job.
-    classifier_data = ndb.JsonProperty(default=None)
```

Comment:
Should we mark this as deprecated, instead of removing this? (This will help us to clean the data in future.)

Reply:
Not necessary, this model is still not launched in prod so there won't be any data that needs to be migrated or cleaned up.